test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/contract/HapiEthereumCall.java

```diff
@@ -267,12 +274,13 @@ protected Consumer<TransactionBody.Builder> opBodyDef(HapiApiSpec spec) throws T
         }
 
         final var longTuple = TupleType.parse("(int64)");
-        final var gasPriceBytes = Bytes.wrap(longTuple.encode(Tuple.of(gasPrice)).array()).toArray();
+        final var gasPriceBytes = Bytes.wrap(longTuple.encode(Tuple.of(gasPrice.longValueExact())).array()).toArray();
```

Comment:
can move `longTuple` up to a class field and utilize with something like 
```suggestion
        final var gasPriceBytes = gasLongToBytes(gasPrice);
```

where 
```java
        private byte[] gasLongToBytes(BigInteger gas) {
		return Bytes.wrap(longTuple.encode(Tuple.of(gas.longValueExact())).array()).toArray();
	}
```